src/Loop.php

```diff
@@ -59,155 +59,162 @@ public static function stop()
     /**
      * Defer the execution of a callback.
      *
-     * @param callable $callback The callback to defer.
+     * @param callable(mixed $data, string $watcherIdentifier) $callback The callback to defer.
```

Comment:
@kelunik Well, I'm fine with either before and after $watcherId, but definitely should be before $cbData